tex/chapters/07-ApplyingModel.tex

```diff
@@ -0,0 +1,197 @@
+%!TEX root = ../main.tex
+
```

Comment:
> An SVM model is defined by many vectors and in our case it uses 596 vectors, each of which is of length 12.

Cool. Include those details and write the vectors to a file that you put somewhere.

Perhaps throw some code together (I'm think it's a couple of lines) to read it in an re use it, to show that it's possible to do so? You could add that as a small section in this chapter: "reusing this model".